### PR TITLE
change UI layout of login page

### PR DESCRIPTION
**Pull request by cociugsergiu**

### CODE REVIEW

File: src/components/Container/index.js [hunk 1]
```diff
@@ -3,7 +3,13 @@ import React from 'react'
 import {Grid} from 'react-flexbox-grid'
 
 const Container = (props) => {
-  return <Grid {...props} style={{paddingLeft: '64px', paddingRight: '64px', width: '1440px'}} />
+  return <Grid {...props} style={
+    {
+      position: props.position,
```
Comment (MihailGorceagEndava) [inline src/components/Container/index.js:8]: should be without position

File: src/features/Header/unauthorized.js [hunk 2]
```diff
@@ -19,7 +19,9 @@ const UnAuthorizedHeader = () => {
 
   const isWhiteColoredHeader = pagesWithWhiteColoredHeader.includes(location.pathname)
   return (
-    <Container>
+    <Container
+      position={location.pathname === LOGIN_PATH ? 'fixed' : ''}
```
Comment (MihailGorceagEndava) [inline src/features/Header/unauthorized.js:23]: no position on container element

File: src/components/Container/index.js (hunk 1 above)
Comment (cociugsergiu) [reply src/components/Container/index.js]: fixed

File: src/features/Header/unauthorized.js (hunk 2 above)
Comment (cociugsergiu) [reply src/features/Header/unauthorized.js]: fixed

**Pull request merged**